docs/packaging-applications/creating-packages/nuget-packages/using-octo.exe.md

```diff
@@ -5,84 +5,75 @@ description: Packaging applications using the octo.exe command line tool for use
 
 In **Octo.exe version 3.3.8** we have expanded the functionality of the `pack` command to allow creating **zip** packages in addition to **NuGet** packages. [Learn more](/docs/packaging-applications/creating-packages/creating-zip-packages.md).
 
-For the purposes of this page, I have created an ASP.NET web site called OctoWeb, and when built the output folder contains all the files required to run my web site.
-
-![](/docs/images/3048097/3277798.png "width=500")
-
-To create a NuGet package of the application, open a command prompt and change directory to where the application is located.
+## Octo Basic Usage
+At a minimum `octo pack` requires an id to be provided:
 
 ```powershell
-C:\>cd Code\OctoWeb\OctoWeb\OctoWeb\bin
+octo pack --id="MyPackage"
 ```
+The above command will generate a package in the current working directory and provide a timestamp based
+version number such as `2018.6.26.190140.nupkg`.
 
-When creating the NuGet package, we will call octo.exe with the pack command and give it a package Id
-
+If you want to provide your own version, you can pass the --version parameter in the call to Octo
 ```powershell
-C:\Code\OctoWeb\OctoWeb\OctoWeb\bin>octo.exe pack --id=OctoWeb
+octo pack --id="MyPackage" --version="1.0.0"
 ```
-
-By default octo.exe will pack all the files that are in the current directory, and give it a timestamp based version number.
-
+You can also change the output directory and folder which will be packed with the `--outFolder` and `--basePath` parameters respectively.
 ```powershell
-C:\Code\OctoWeb\OctoWeb\OctoWeb\bin> octo.exe pack --id=OctoWeb
-Octopus Deploy Command Line Tool, version 3.3.8+Branch.master.Sha.f8a34fc6097785d7d382ddfaa9a7f009f29bc5fb
-
-Packing OctoWeb version 2016.3.9.123003...
-Saving OctoWeb.2016.3.9.123003.nupkg to C:\Code\OctoWeb\OctoWeb\OctoWeb\bin...
-Done.
-C:\Code\OctoWeb\OctoWeb\OctoWeb\bin> dir *.nupkg
-
-    Directory: C:\Code\OctoWeb\OctoWeb\OctoWeb\bin
-
-Mode                LastWriteTime     Length Name
-----                -------------     ------ ----
--a---         9/03/2016  12:30 PM   43628908 OctoWeb.2016.3.9.123003.nupkg
-
+octo pack --id="MyPackage" --version="1.0.0" --basePath="folder/to/pack" --outFolder="destination/folder/path"
 ```
 
-If you want to provide your own version, you can pass the --version parameter in the call to Octo.exe
+## Packaging a .NET Core application
+The process for packaging a .NET core applications is pretty straightforward and can usually be achieved by publishing the application and then using `octo pack` on the resultant output folder for example:
 
 ```powershell
-C:\Code\OctoWeb\OctoWeb\OctoWeb\bin> octo.exe pack --id=OctoWeb --version=1.0.0.0
-Octopus Deploy Command Line Tool, version 3.3.8+Branch.master.Sha.f8a34fc6097785d7d382ddfaa9a7f009f29bc5fb
+dotnet publish ./OctoWeb.csproj --output ./dist
+octo pack ./dist --id="OctoWeb" --version="1.0.0"
```

Comment:
should this line be `dotnet octo pack...` ?

Reply:
shouldn't it be `octo.exe pack...`

we just need them to publish to dotnet don't we, then it's handover to octo.exe?

Reply:
ah sry, so this is the dotnetcore version, gotcha

Reply:
not to be confused with dotnet pack :crazy:

Reply:
This was my reasoning as well, but if we want this to align to what we are about to publish in the blog and how we want people to use it with dotnet core it may make sense to use `dotnet octo pack` instead. It is going to be way easier if you already have dotnet to just do `dotnet tool install Octopus.DotNet.Cli --global` and you are ready to rock...

Reply:
Actually that last bit may trip some people up, they may think `dotnet pack` == `dotnet octo pack`

Reply:
yea I think a callout below this to specifically state "pls note `dotnet octo pack !== dotnet pack` thx k bye"